README.md

```diff
@@ -1,6 +1,18 @@
 # Kaoto examples
 Hosting example integrations provided by users
 
+# How to use
+There are a few ways to use these examples:
+
+## [Red Hat Dev Spaces](https://workspaces.openshift.com/#https://github.com/KaotoIO/kaoto-examples.git)
+This is the easiest way to get started with the examples. It also provides a full development environment with all the tools you need to develop and test your integrations.
+
+## [vscode.dev](https://vscode.dev/github/KaotoIO/kaoto-examples)
+This is a lightweight, browser-based version of Visual Studio Code that lets you develop entirely in the browser. It's a great way to quickly check out the examples.
+
+## Clone the repository and run the examples locally
+Alternatively, you could clone the repository and run the examples locally. To do this, you will need to have the Camel CLI installed. [You can find instructions on how to install the Camel CLI here](https://camel.apache.org/manual/camel-jbang.html).
```

Comment:
I am wondering if we should rather link to the quickstart guide on Kaoto.io (which has indirectly also the link to the Camel CLI) -> https://kaoto.io/docs/quickstart/

Reply:
Yup, I can do that then

Reply:
Done